types/office-js/index.d.ts

```diff
@@ -34227,7 +34227,7 @@ declare namespace Excel {
          * @param key The key of the new setting.
          * @param value The value for the new setting.
          */
-        add(key: string, value: string | number | boolean | Date | any[] | any): Excel.Setting;
+        add(key: string, value: string | number | boolean | Date | Array<any> | any): Excel.Setting;
```

Comment:
build is failing but gives suggestion to keep `any[]` instead of using `Array<any>`

```suggestion
        add(key: string, value: string | number | boolean | Date | any[] | any): Excel.Setting;
```